### DISCUSSION TITLE
Apps stuck on the free plan

### DISCUSSION THREAD

**Discussion by jimmedan** [supabase/supabase#31165] [Questions]
I have two apps developed on the Free Supabase plan, but I have subsequently paid for the pro plan; how do I move my free plan apps over to my paid plan

**Comment by GaryAustin1** [comment]
You can transfer them to your paid org from the free org.  BUT you will be charged for them for compute time (will share egress,etc.) at a minimum of $10 a month.
![image](https://github.com/user-attachments/assets/80029377-78dc-4bea-9e6a-49efb347d3d7)
